README.md

```diff
@@ -263,7 +263,8 @@ plugin_packages
 
 There are some known limitations when installing a package:
 
-- Different plugins cannot depends on different/incompatible modules. If plugin A require module x at version 1 and plugin B require the same module X at version 2 then plugin A and plugin B cannot be installed simultaneously. Version 2 of module X will be used and this can cause problems on your code.
+- Different plugins cannot depend on different/incompatible modules. If plugin A require module x at version 1 and plugin B require the same module X at version 2 then plugin A and plugin B cannot be installed simultaneously. Version 2 of module X will be used and this can cause problems on your code.
```

Comment:
Minor typofix here :)

Reply:
Thank you! my English is bad :flushed: , any fix is welcome!!

Reply:
Non è un problema! Sto imparando l'italiano quindi so come ci si sente!